apps/server-monolith/app/controllers/concerns/current_model.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+## Set current_user and current config before action
+module CurrentModel
```

Comment:
`CurrentModel` should be renamed to `CurrentContextSetter`, because it actually contains `current_model` and `current_config` setters.